wire-library/wire-runtime/src/jvmMain/kotlin/com/squareup/wire/CamelCase.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire
+
+// This mirrors the logic of `fieldNameToJsonName` in `com/google/protobuf/Descriptors.java`, except
+// it lower case the first character as well (protoc seems to do this step earlier).
+// See https://github.com/protocolbuffers/protobuf/blob/master/java/core/src/main/java/com/google/protobuf/Descriptors.java
+fun camelCase(string: String): String {
+  return buildString {
```

Comment:
It'll never be larger so might as well
```suggestion
  return buildString(string.length) {
```

Reply:
fixed